README.md

```diff
@@ -60,4 +60,19 @@ Once built, you can run remote commands like this:
 ./winrm -hostname remote.domain.com -username "Administrator" -password "secret" "ipconfig /all"
 ```
 
+## Docker
+
+### Build container
+
+```
+docker build -t winrm .
+```
+
+### Usage
+
+Once built, you can run remote commands like this:
+
+```sh
+docker run -it --rm winrm winrm -hostname remote.domain.com -username "Administrator" -password "secret" "ipconfig /all"
```

Comment:
Given that the image is meant to run the winrm client, one could set the winrm binary as the image's entrypoint.